workers_launch_template.tf

```diff
@@ -190,7 +88,7 @@ resource "aws_autoscaling_group" "workers_launch_template" {
   }
 
   dynamic "warm_pool" {
-    for_each = lookup(var.worker_groups_launch_template[count.index], "warm_pool", null) != null ? [lookup(var.worker_groups_launch_template[count.index], "warm_pool")] : []
+    for_each = lookup(var.worker_groups_launch_template[each.key], "warm_pool", null) != null ? [lookup(var.worker_groups_launch_template[each.key], "warm_pool")] : []
```

Comment:
```suggestion
    for_each = lookup(each.value, "warm_pool", null) != null ? [lookup(each.value, "warm_pool")] : []
```

Reply:
fixed multiple occurrences of the same pattern :heavy_check_mark: